.github/workflows-source/schedule-daily.yml

```diff
@@ -179,7 +179,6 @@ jobs:
         uses: ./.github/actions/bazel-test-all/
         with:
           BAZEL_COMMAND: "test"
-          BAZEL_TARGETS: "//rs/nns/..."
```

Comment:
Note to reviewers: this is to ensure that tests tagged with `nns_tests_nightly` outside the `/rs/nns` also run nightly.  Note that `BAZEL_TARGETS` defaults to `//...`.